src/PrestaShopBundle/Resources/views/Admin/Layout/legacy_layout.html.twig

```diff
@@ -34,7 +34,15 @@
   {% endblock %}
 </head>
 
-<body class="lang-{{ ps.isoUser }}{% if ps.isRtlLanguage %} lang-rtl{% endif %} ps_back-office page-sidebar {{ ps.controllerName|escape|lower }}{% if ps.menuCollapsed %} page-sidebar-closed{% endif %}{% if ps.multiShop|default(false) %} multishop-enabled{% endif %}{% if ps.debugMode|default(false) %} developer-mode{% endif %}"
+{# Construct CSS classes based on version numbers #}
+{% set major_version = constant('PrestaShop\\PrestaShop\\Core\\Version::MAJOR_VERSION') %}
```

Comment:
Same as for the previous comment, better ask core dev to do it on the back end.

Reply:
See previous comment in default layout